index.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+const Hp = require('hemera-plugin')
+const AWS = require('aws-sdk')
+const DynamoStore = require('./store')
+const StorePattern = require('hemera-store/pattern')
+const Joi = require('joi')
```

Comment:
You don't have to include it. You will depend on `hemera-joi` and you can use `const joi = hemera.joi`